src/style.css

```
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: 400;
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 900;
+  src: url("fonts/Avenir.ttc");
+}
+
+.searchbar {
+  width: 100%;
+  position: relative;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-color: #fff;
+  margin-top: 20px;
+}
+
+.searchbar__label {
+  display: block;
+  height: 68px;
+}
+
+.small_label {
+  height: 40px;
+}
+
+.searchbar__input {
+  width: 100%;
+  height: 99%;
+  padding-left: 62px;
+  outline: none;
+  border: none;
+  font-family: Avenir, sans-serif;
+  font-weight: 900;
+  font-size: 16px;
+}
+
+.searchbar__input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.small_input {
+  padding-left: 33px;
+  font-family: Avenir, sans-serif;
+  font-weight: 900;
+  font-size: 14px;
+}
+
+.small_input::placeholder {
+  font-family: Avenir, sans-serif;
```

You already have this font-family in searchbar__input
```suggestion
  font-weight: 400;
  font-size: 16px;
}

.small_input {
  padding-left: 33px;
  font-size: 14px;
}

.small_input::placeholder {
```